src/service/auth/signup.ts

```
@@ -0,0 +1,50 @@
+import { Request, Response } from 'express';
+import { prisma } from '../../config/prisma';
+import { checkMailRegex, checkPasswordRegex } from '../../utils/regex';
+import bcrypt from 'bcrypt';
+import { SignUpRequest } from '../../types/auth';
+
+export const signUp = async (req: Request<{}, {}, SignUpRequest>, res: Response) => {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)